### DISCUSSION TITLE
What about Windows?

### DISCUSSION THREAD

**Discussion by brettforbes** [mitre/caldera#3058] [General]
Hi,

Your software looks amazing, great stuff.

I notice it compiles on Mac and Linux, but not Windows.

Is there a plan to migrate it to Windows, or is there some reason that makes it impossible?

Am I being dumb asking for this? If so sorry

**Comment by clenk** [comment]
Thanks for the question (not dumb!). Just to be clear, Caldera has a client/server architecture. The server is only tested on Linux and Mac, but the [client agents](https://github.com/mitre/sandcat) are available for Linux, Mac, _and_ Windows. Testing and fixing server compatibility with Windows has not been a priority request thus far.

**Reply by brettforbes** [reply]
Thank you for the clarity, thats super helpful.

We are working on a use case that we believe will be of high value to you guys, but it is still a few months off.

We can Virtual Box initially, and then work with you guys to see if there is a case to go native. All good, thank you

**Reply by clenk** [reply]
Sounds good. Keep us posted; would love to hear or see what you're doing with it!
